apps/dashboard/app/javascript/packs/config.js

```diff
@@ -38,11 +38,34 @@ function navbarHighlight(theme, color) {
     }`;
 }
 
-function getConfigData() {
+function maxFileSize () {
   const cfgData = configData();
 
-  return cfgData;
+  // Check if cfgData['maxFileSize'] is just empty string, 
+  // if so set default of maxFileUpload=10737420000 bytes.
+  if (cfgData['maxFileSize'].length == 0) {
+    return parseInt(10737420000, 10);
```

Comment:
The issue looks to be it will pass a string back if the file size is being set by a config, so you need to cast it using `parsInt` regardless in that case.

Reply:
I don't know if I follow.  I get you need to cast the string to an integer in the `else` block, but in the `if` block, can't you just return the integer directly?

```javascript
    return 10737420000;
```

![image](https://user-images.githubusercontent.com/4874123/217302468-fb422371-e3bb-4953-890d-41691c3370a5.png)